### ISSUE TITLE
Unable to run a newly created project.

### ISSUE THREAD

**Issue by GeneralNitin** [SAP/fundamental-ngx#6117]
I followed the all the steps mentioned in the Full Installation Guide.
There are some issues, I am facing while executing the project.

1. The Generate SCSS is giving an error.
2. After commenting the content of styles.scss file , I get another error.

**Logs:**

1.
```
./src/styles.scss - Error: Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleError: Module Error (from ./node_modules/postcss-loader/dist/cjs.js):
(Emitted value instead of an instance of Error) CssSyntaxError: /Users/Nitin/LocalDisk C/Work/Angular/Fundamental/src/styles.scss:4:4: Can't resolve '@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Regular-full.woff' in '/Users/Nitin/LocalDisk C/Work/Angular/Fundamental/src'

  2 | @font-face {
  3 |     font-family: '72';
> 4 |     src: url('~@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Regular-full.woff') format('woff');
    |    ^
  5 |     font-weight: normal;
  6 |     font-style: normal;

    at Object.emitError (/Users/Nitin/LocalDisk C/Work/Angular/Fundamental/node_modules/webpack/lib/NormalModule.js:550:6)
    at Declaration (/Users/Nitin/LocalDisk C/Work/Angular/Fundamental/node_modules/@angular-devkit/build-angular/src/webpack/plugins/postcss-cli-resources.js:144:28)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async LazyResult.runAsync (/Users/Nitin/LocalDisk C/Work/Angular/Fundamental/node_modules/postcss/lib/lazy-result.js:411:15)
    at async Object.loader (/Users/Nitin/LocalDisk C/Work/Angular/Fundamental/node_modules/postcss-loader/dist/index.js:97:14)
```

2.
```
Error: node_modules/@fundamental-ngx/core/node_modules/@angular/platform-browser/platform-browser.d.ts:703:5 - error TS2416: Property 'supportsDOMEvents' in type 'ɵangular_packages_platform_browser_platform_browser_o' is not assignable to the same property in base type 'ɵDomAdapter'.
  Type '() => boolean' is not assignable to type 'boolean'.

703     supportsDOMEvents(): boolean;
        ~~~~~~~~~~~~~~~~~


Error: node_modules/@fundamental-ngx/core/node_modules/@angular/platform-browser/platform-browser.d.ts:703:5 - error TS2425: Class 'ɵDomAdapter' defines instance member property 'supportsDOMEvents', but extended class 'ɵangular_packages_platform_browser_platform_browser_o' defines it as instance member function.

703     supportsDOMEvents(): boolean;
```

**Comment by kavya-b** [comment]
Hi @GeneralNitin , could you try replacing the font-face content in `styles.scss` with this: 
```
@font-face {
    font-family: "72";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Regular-full.woff")
        format("woff");
    font-weight: normal;
    font-style: normal;
}

@font-face {
    font-family: "72";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Light.woff")
        format("woff");
    font-weight: 300;
    font-style: normal;
}

@font-face {
    font-family: "72";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Bold.woff")
        format("woff");
    font-weight: 700;
    font-style: normal;
}

@font-face {
    font-family: "SAP-icons";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/SAP-icons.woff")
        format("woff");
    font-weight: normal;
    font-style: normal;
}
```

the underlying `theming-base-content` package's path has changed a bit recently, so this might be the issue. 
Could you let us know if this change fixed your issue?

**Comment by GeneralNitin** [comment]
Hi @kavya-b,

The first error is now fixed, which was related to SCSS.
However the second one still exists.

```
Error: node_modules/@fundamental-ngx/core/node_modules/@angular/platform-browser/platform-browser.d.ts:703:5 - error TS2416: Property 'supportsDOMEvents' in type 'ɵangular_packages_platform_browser_platform_browser_o' is not assignable to the same property in base type 'ɵDomAdapter'.
  Type '() => boolean' is not assignable to type 'boolean'.

703     supportsDOMEvents(): boolean;
        ~~~~~~~~~~~~~~~~~


Error: node_modules/@fundamental-ngx/core/node_modules/@angular/platform-browser/platform-browser.d.ts:703:5 - error TS2425: Class 'ɵDomAdapter' defines instance member property 'supportsDOMEvents', but extended class 'ɵangular_packages_platform_browser_platform_browser_o' defines it as instance member function.

703     supportsDOMEvents(): boolean;
``` 

Please let me know a fix for this.
Setup:
Browser: Google Chrome
Editor: VS-Code
Angular CLI: 12.1.4
Node: 14.17.3
Package Manager: npm 6.14.13
OS: darwin x64

Angular: 12.1.4
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1201.4
@angular-devkit/build-angular   12.1.4
@angular-devkit/core            12.1.4
@angular-devkit/schematics      12.1.4
@schematics/angular             12.1.4
rxjs                            6.6.7
typescript                      4.3.5

**Comment by kavya-b** [comment]
@droshev any ideas on this issue? I can reproduce it as follows:

- in a new ang12 project, run `ng add @fundamental-ngx/core`
- go to `app.module.ts` and add any core module, eg, `CheckboxModule`
- run `ng build` --> error

![image](https://user-images.githubusercontent.com/53509521/127516887-249b922d-0c6c-429f-8412-e7d9182f0743.png)

I tried changing the typescript version to a lower one(same as ngx), but ang12 does not support that version. The lowest allowed version (4.2.3) also throws same error on building. I also verified on ang10 and ang11 and this issue is not reproducible there, so this may be ang12 specific.
